connectivity/connectivity.go

```diff
@@ -32,8 +37,26 @@ func Make(cfg *configuration.Observer, obs *observability.Observability) *Connec
 			)
 			log.Infof("trying connect to %s...", cfg.Replicator.Addr)
 
-			// We omit error here because connect happens in background.
-			conn, err := grpc.Dial(cfg.Replicator.Addr, limits, grpc.WithInsecure())
+			cp, err := x509.SystemCertPool()
+			if err != nil {
+				log.Fatal(errors.Wrapf(err, "failed get x509 SystemCertPool"))
+			}
+			httpClient := &http.Client{Transport: &http.Transport{
```

Comment:
Maybe we should set up `http.Client` and `http.Transport` timeouts to prevent idle connections?

Reply:
Good idea. What do you think, should we pass these parameters through a configuration?

Reply:
Yeap, I think so.

Reply:
I've added a common timeout for an HTTP client.